### ISSUE TITLE
PLANET-5253 Footer Accessibility

### ISSUE THREAD

**Issue by planet-4** [greenpeace/planet4#87]
The footer is generally ok, it just needs some tiny tweaks:

* The slashes are read out loud as "diagonal", they should be *aria-hidden*.
* The copyright text should be wrapped by an element with *role=text* to be treated as a single text.
* Same with the Copyright symbol and the NRO text.

```
Reporter: pcuadrad
Sections: Footer
```
Potentially affected repositories: [planet4-master-theme](https://github.com/greenpeace/planet4-master-theme)

**Comment by comzeradd** [comment]
Fixed at https://github.com/greenpeace/planet4-master-theme/pull/1219
